web/src/OverviewTabBar.tsx

```diff
@@ -2,29 +2,110 @@ import React, { useEffect, useState } from "react"
 import { matchPath } from "react-router"
 import { Link, useHistory } from "react-router-dom"
 import styled from "styled-components"
+import { ReactComponent as CloseSvg } from "./assets/svg/close.svg"
+import { ReactComponent as LogoWordmarkSvg } from "./assets/svg/logo-wordmark-gray.svg"
 import { useLocalStorageContext } from "./LocalStorage"
-import { Color } from "./style-helpers"
+import {
+  AnimDuration,
+  Color,
+  ColorAlpha,
+  ColorRGBA,
+  Font,
+  FontSize,
+  SizeUnit,
+} from "./style-helpers"
 
 type OverviewTabBarProps = {
   tabsForTesting?: string[]
 }
 
 let OverviewTabBarRoot = styled.div`
+  padding-top: ${SizeUnit(0.5)};
   display: flex;
   width: 100%;
-  height: 68px;
-  background-color: ${Color.gray};
+  height: 60px;
+  box-sizing: border-box;
+  background-color: ${Color.grayDarkest};
   border-bottom: 1px solid ${Color.grayLight};
+  align-items: stretch;
 `
 
 export let Tab = styled(Link)`
+  cursor: pointer;
   display: flex;
-  border: 1px solid ${Color.grayLight};
-  border-radius: 4px 4px 0px 0px;
-  margin: 12px;
+  font-family: ${Font.sansSerif};
+  font-size: ${FontSize.smallest};
   flex-grow: 0;
-  padding: 8px;
+  color: ${Color.grayLightest};
+  align-items: stretch;
   text-decoration: none;
+  padding-left: ${SizeUnit(0.5)};
+  box-sizing: border-box;
+  // "Remove" the Tab's bottom border (i.e., OverviewTabBarRoot's border)
+  margin-bottom: -1px;
+  // Define borders now so it doesn't "jump" when selected
+  border-top: 1px solid ${ColorRGBA(Color.black, 0)};
+  border-left: 1px solid ${ColorRGBA(Color.black, 0)};
+  border-right: 1px solid ${ColorRGBA(Color.black, 0)};
+
+  transition: background-color ${AnimDuration.short} linear,
+    color ${AnimDuration.default} linear,
+    border-color ${AnimDuration.short} linear;
+
+  &:focus {
+    outline: 0;
+    border-top-color: ${Color.gray};
+  }
+
+  &:hover {
+    color: ${Color.white};
+  }
+
+  &.isSelected {
+    color: ${Color.white};
+    background-color: ${Color.grayDark};
+    border-top-color: ${Color.grayLight};
+    border-left-color: ${Color.grayLight};
+    border-right-color: ${Color.grayLight};
+    border-radius: 4px 4px 0px 0px;
+  }
+`
+
+let TabName = styled.div`
+  display: flex;
+  align-items: center;
+`
+
+export let HomeTab = styled(Link)`
+  border: none;
+  padding-left: ${SizeUnit(1)};
+  padding-right: ${SizeUnit(1)};
+  background-color: transparent;
+  display: flex;
+  align-items: center;
+  opacity: ${ColorAlpha.almostOpaque};
+  transition: opacity ${AnimDuration.short} ease;
+
+  &:hover {
+    opacity: 1;
+  }
+`
+
+let CloseButton = styled.button`
+  cursor: pointer;
+  background-color: transparent;
+  border: 0 none;
+  padding-left: ${SizeUnit(0.25)};
+  padding-right: ${SizeUnit(0.25)};
+
+  svg {
+    fill: none;
```

Comment:
i'm surprised you used fills, rather than, say, transitioning the opacity, but this is ok too